content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -3,7 +3,7 @@ reviewers:
 - janetkuo
 title: TTL Controller for Finished Resources
 content_template: templates/concept
-weight: 65
+weight: 70
```

Comment:
nit: Check the weight variables in `ttlafterfinished.md` and `garbage-collection.md`.